tests/benchmark/MutationGenerator/generate-mutations-closure.php

```diff
@@ -65,7 +65,8 @@ static function (SplFileInfo $fileInfo): Trace {
 );
 
 $mutators = $container->getMutatorFactory()->create(
-    $container->getMutatorResolver()->resolve(['@default' => true])
+    $container->getMutatorResolver()->resolve(['@default' => true]),
+    true
```

Comment:
This parameter was missing and crashing my local e2e tests (`make test`).

Something about no-op mutators.